debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/JavaVersionHelper.java

```diff
@@ -25,30 +25,50 @@
 
 import com.ibm.j9ddr.CorruptDataException;
 import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
-import com.ibm.j9ddr.vm29.types.UDATA;
 
 /**
  * JavaVersionHelper helps check if the JVM version is new enough for the modularity DDR commands
  */
 public class JavaVersionHelper 
 {
-	public final static int J2SE_SERVICE_RELEASE_MASK = 0xFFFF;
-	public final static int J2SE_19 = 9;
-	public final static int J2SE_JAVA_SPEC_VERSION_SHIFT = 8;
+	private final static int J2SE_SERVICE_RELEASE_MASK = 0xFFFF;
+	private final static int J2SE_19 = 9;
+	private final static int J2SE_JAVA_SPEC_VERSION_SHIFT = 8;
 	
 	/**
 	 * Returns true if the Java version is Java9 and up.
+	 * @deprecated Use {@link #ensureMinimumJavaVersion(int, J9JavaVMPointer, PrintStream)} instead.
```

Comment:
I'm not sure it matters, but `@deprecated` normally appears at the end of the javadoc (i.e. after line 44).